### PR TITLE
Fixes issue with sub modules

### PR DESCRIPTION
**Pull request by mikevalstar**
fixes the following issue with sub modules

https://github.com/therealklanni/git-guppy/issues/39

### CODE REVIEW

File: bin/index.js [hunk 1]
```diff
@@ -40,7 +42,16 @@ if (argv.dest) {
 } else {
   var topLevel = exec('git rev-parse --show-toplevel', { silent: true })
     .output.slice(0, -1);
-  dest = topLevel + '/.git/hooks/';
+  if(test('-f', topLevel + '/.git')){
+    // this is a sub module
+    var buf = fs.readFileSync(topLevel + '/.git', "utf8").trim();
+    if(buf.substr(0,6) == 'gitdir'){
```
Comment (therealklanni) [inline bin/index.js]: Please follow existing style (spaces after keywords and before opening curly), and use `===` only.

File: bin/index.js [hunk 2]
```diff
@@ -40,7 +42,16 @@ if (argv.dest) {
...
+      topLevel = topLevel + '/' + buf.substr(8).trim();
+    }
+    dest = topLevel + '/hooks/';
+  }else{
```
Comment (therealklanni) [inline bin/index.js]: Space around keyword

**Comment by mikevalstar** [comment]
The styling has been updated

File: bin/index.js [hunk 3]
```diff
@@ -40,7 +42,16 @@ if (argv.dest) {
 } else {
   var topLevel = exec('git rev-parse --show-toplevel', { silent: true })
     .output.slice(0, -1);
-  dest = topLevel + '/.git/hooks/';
+  if(test('-f', topLevel + '/.git')){
```
Comment (therealklanni) [inline bin/index.js]: Please use space around keywords (`if`) and before opening curlies.

File: bin/index.js [hunk 4]
```diff
@@ -40,7 +42,16 @@ if (argv.dest) {
...
+    // this is a sub module
+    var buf = fs.readFileSync(topLevel + '/.git', "utf8").trim();
+    if(buf.substr(0,6) === 'gitdir'){
```
Comment (therealklanni) [inline bin/index.js]: `if` and curly again :curly_loop:

**Comment by therealklanni** [comment]
Thanks. Just a couple more style issues before I can merge.

**Comment by mikevalstar** [comment]
Alright; additional changes made.

**Comment by therealklanni** [comment]
:metal: Thanks! Sorry for the delay.

**Comment by mikevalstar** [comment]
No problem; going to start using this in all my projects now ;)

**Comment by therealklanni** [comment]
Awesome! :+1:

**Pull request merged**